ical/parsing/parser.py

```diff
@@ -67,7 +67,7 @@ def _create_parser() -> ParserElement:
     param_name = Or([iana_token, x_name])
     # rfc5545 Q-SAFE-CHAR is any character except CONTROL and DQUOTE which is
     # close enough to the pyparsing provided parser element
-    param_text = Word(SAFE_CHAR)
+    param_text = Word(SAFE_CHAR) | ""
```

Comment:
Is this equivalent to the `Or` syntax above? If so please use that so its consistent with the existing code.

Reply:
@allenporter : It is not exactly equivalent, this is actually making it `Opt` (Optional). I could change it to `Opt(Word(SAFE_CHAR))` if you would prefer (and add the Opt import above).

Reply:
Thanks i see this in the doc. We're also using other expression operators above like "+" so this seems fine.